tree-construction/doctype01.dat

```diff
@@ -472,3 +475,43 @@
 | <html>
 |   <head>
 |   <body>
+
+#data
+<!DOCTYPE html SYSTEM "http://www.ibm.com/data/dtd/v11/ibmxhtml1-transitional.dtd"><p><table>
```

Comment:
All of these are special cases for quirks mode that were not covered. The `<p><table>` at the end ensures we do end up in quirks mode.